sources/USAGE.md

```diff
@@ -20,7 +20,7 @@ allow to enable the old Tableaux-like SAT-solver:
 
 ### Files extensions
 Alt-Ergo supports file extensions:
-- `.why`, `.mlw`, `.ae` for its native input language
+- `ae`, for its native input language, `why` and `mlw` are depreciated
```

Comment:
```suggestion
- `.ae`, for its native input language (`.why` and `.mlw` are now depreciated although still accepted)
```
Added . in front of extensions and a better message for the depreciation of .why and .mlw files (I'm not really sure about my new version of the message though, it's mostly a draft)